drivers/sdhc/sdhc_esp32.h

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define SDMMC_FREQ_DEFAULT   20000 /*!< SD/MMC Default speed (limited by clock divider) */
+#define SDMMC_FREQ_HIGHSPEED 40000 /*!< SD High speed (limited by clock divider) */
+#define SDMMC_FREQ_PROBING   400   /*!< SD/MMC probing speed */
+#define SDMMC_FREQ_52M       52000 /*!< MMC 52MHz speed */
+#define SDMMC_FREQ_26M       26000 /*!< MMC 26MHz speed */
+
+/* SD commands remapping */
+#define MMC_WRITE_BLOCK_MULTIPLE SD_WRITE_MULTIPLE_BLOCK
+#define MMC_READ_BLOCK_MULTIPLE  SD_READ_MULTIPLE_BLOCK
+#define MMC_WRITE_DAT_UNTIL_STOP 20
+#define MMC_READ_DAT_UNTIL_STOP  11
+#define MMC_STOP_TRANSMISSION    SD_STOP_TRANSMISSION
+#define MMC_GO_IDLE_STATE        SD_GO_IDLE_STATE
+#define MMC_APP_CMD              SD_APP_CMD
+
+const uint32_t SDMMC_DATA_ERR_MASK = SDMMC_INTMASK_DTO | SDMMC_INTMASK_DCRC | SDMMC_INTMASK_HTO |
```

Comment:
Should be `const static` if being defined in a header file- but why can't these be defined using `#define`?